src/Apps/Auction/Components/Form/AddressForm.tsx

```diff
@@ -80,7 +127,7 @@ export const AddressForm = () => {
         />
       </Column>
 
-      <Column span={6}>
+      <Column span={12}>
```

Comment:
Defaults to 12

Reply:
When I removed this the layout broke so I think the 12 is actually required. Replaced in [4fa5922](https://github.com/artsy/force/pull/14846/commits/4fa5922cd24b93d4bf9b7ffb3b92404b65e2e941).